src/library/clsparse-init.cpp

```diff
@@ -106,6 +106,7 @@ cldenseInitMatrix( cldenseMatrix* denseMatx )
     return clsparseSuccess;
 }
 
+// To be removed?
```

Comment:
Yes please, it's a dead function.  But the real coo2csr exports need to have CLSPARSE_EXPORT decorated in clsparse-c002csr-GPU.cpp